src/parser/open-api/v3/open-api-v3-validator.js

```diff
@@ -6,16 +6,34 @@ const VALIDATOR_HOST = 'https://openapi-converter.herokuapp.com'
  * @param {string} definitionUrl
  * @return {Promise<boolean>}
  */
-export async function validateDefinition (definitionUrl) {
+export async function validateDefinition (definition) {
+  const headers = new Headers({
+    'Accept': 'application/json'
+  })
+
+  const form = new FormData()
+
+  form.append('source', definition)
+
   const url = `${VALIDATOR_HOST}/api/v1/validate`
-  const result = await fetch(`${url}?url=${definitionUrl}`)
+  const fetchOptions = {
+    method: 'POST',
+    headers: headers,
+    body: form
+  }
+  const result = await fetch(`${url}`, fetchOptions)
+
+  if (result.ok === false) {
+    throw new Error(`There was a problem with the validation service, please see the console and ${VALIDATOR_HOST}.`)
+  }
 
-  if (result.ok) {
-    const response = await result.json()
+  // TODO change this to JSON parsing when
+  // https://github.com/Mermade/openapi-webconverter/issues/2 is resolved
```

Comment:
Looks like this issue is fixed now :)

Reply:
Yep, just had to change some test cases.